React/Views/RCTTextField.m

```diff
@@ -42,6 +42,25 @@ - (void)setText:(NSString *)text
   }
 }
 
+- (void)_setupPlaceholder {
```

Comment:
To be clear, that's what *should* happen - I believe in the current implementation it would continue to show the previous placeholder value, which is incorrect.